model_compression_toolkit/qat/keras/qat_model_builder.py

```diff
@@ -54,6 +47,22 @@ def _is_qat_applicable(node: common.BaseNode,
     return fw_info.is_kernel_op(node.type) and node.is_weights_quantization_enabled()
 
 
+def qat_wrapper(n, layer):
+    """
+    A function which takes a computational graph node and a keras layer and perform the quantization wrapping
+    Args:
+        n: A node of mct graph.
+        layer: A keras layer
+
+    Returns: Wrapped layer
+
+    """
+    if _is_qat_applicable(n, DEFAULT_KERAS_INFO):
+        return qi.KerasQuantizationWrapper(layer, quantization_dispatcher_builder(n, DEFAULT_KERAS_INFO))
```

Comment:
Since the dispatcher builder is per framework, let's use KERAS_INFO inside of the builder, instead of passing it